engines/stark/services/userinterface.h

```diff
@@ -165,6 +165,11 @@ class UserInterface {
 	/** Directly open or close a screen */
 	void toggleScreen(Screen::Name screenName);
 
+	/** Toggle subtitles on and off */
+	void requestToggleSubtitle() { _toggleSubtitle = !_toggleSubtitle; }
+	bool hasToggleSubtitleRequest() { return _toggleSubtitle; }
+	void performToggleSubtitle();
```

Comment:
Based on the original game, the toggling of subtitles will be performed after the finishing of the current speech. So some kind of delay needs to be done here.